src/dex/hashflow/hashflow.ts

```diff
@@ -726,6 +726,9 @@ export class Hashflow extends SimpleExchange implements IDex<HashflowData> {
             `${this.dexKey}-${this.network}: Market Maker ${mm} failed to build transaction on side ${side} with too strict slippage. Skipping restriction`,
           );
         } else {
+          this.logger.error(
```

Comment:
let's keep it warn (same as most of the errors and [this log](https://github.com/paraswap/paraswap-dex-lib/blob/0b688cd16727616b16c26130adc5d19a15f07792/src/dex/hashflow/hashflow.ts#L741)).

Reply:
done